config/boards/rockpi-s.conf

```diff
@@ -2,5 +2,4 @@
 BOARD_NAME="ROCK Pi S"
 BOARDFAMILY="rockpis"
 BOOTCONFIG="rockpi-s-rk3308_defconfig"
-KERNEL_TARGET="legacy"
-
+KERNEL_TARGET="legacy,current,dev"
```

Comment:
I would propose to disable "current" here.

Reply:
I'm not extremely familiar with the build system - will disabling it here will still let you select the `current` branch from build script options? 
Basically this was my stop-gap solution while moving over from my branch to a patches

Reply:
> will disabling it here will still let you select the current branch from build script options?

No

> It is pretty usable already with console, storage and networking working.

The idea of DEV branch is to be in any, usually partially working, state. Current is "actual stable branch" and if this is so, I have not complains and we keep as is. Even its tied to non LTS kernel ATM.